airbrakes/logger.py

```diff
@@ -35,14 +37,32 @@ def __init__(self):
             writer = csv.DictWriter(file_writer, fieldnames=CSV_HEADERS)
             writer.writeheader()
 
-        # Makes a queue to store log messages, basically it's a process-safe list that you add to the back and pop from
-        # front, meaning that things will be logged in the order they were added
-        self.log_queue = multiprocessing.Queue()
-        self.running = multiprocessing.Value("b", True)  # Makes a boolean value that is shared between processes
+        # Makes a queue to store log messages, basically it's a process-safe list that you add to
+        # the back and pop from front, meaning that things will be logged in the order they were
+        # added
+        self._log_queue: multiprocessing.Queue[IMUDataPacket] = multiprocessing.Queue()
```

Comment:
Shouldn't this type be a Queue of dictionaries?

Reply:
yes thanks should be updated